first_hour_on_terra/workspace_resource_examples.ipynb

```diff
@@ -0,0 +1,1552 @@
+    "                                         f\"--description={description_content}\",\n",
+    "                                         f\"--name={self.input_name.value}\",\n",
+    "                                         f\"--workspace={CURRENT_WORKSPACE_ID}\"],\n",
+    "                                        capture_output=True,\n",
+    "                                        text=True)\n",
+    "                print(result.stderr) if not result.stdout else print(\n",
+    "                    result.stdout)\n",
+    "            else:\n",
+    "                terraCommand = f\"\"\"terra resource create bq-dataset \\\\\n",
+    "                --access={self.access_drop_down.value} \\\\\n",
+    "                --cloning={self.cloning_drop_down.value} \\\\\n",
+    "                --description={description_content} \\\\\n",
+    "                --name={self.input_name.value} \\\\\n",
+    "                --workspace={CURRENT_WORKSPACE_ID}\"\"\"\n",
+    "                print(terraCommand)\n",
+    "                result = subprocess.run([\"terra\", \"resource\", \"create\", \"bq-dataset\",\n",
+    "                                         f\"--access={self.access_drop_down.value}\",\n",
+    "                                         f\"--cloning={self.cloning_drop_down.value}\",\n",
+    "                                         f\"--description={description_content}\",\n",
+    "                                         f\"--name={self.input_name.value}\",\n",
+    "                                         f\"--workspace={CURRENT_WORKSPACE_ID}\"],\n",
+    "                                        capture_output=True,\n",
+    "                                        text=True)\n",
+    "                print(result.stderr) if not result.stdout else print(\n",
+    "                    result.stdout)\n",
+    "\n",
+    "\n",
+    "create_bq_dataset_widget = CreateBQDataset()\n",
+    "display(create_bq_dataset_widget.vb)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "#### 2. Create a Google Cloud Storage bucket\n",
+    "\n",
+    "Run the cell below to create a widget, then populate the widget's input fields and click the button to add a Google Cloud Storage bucket as a controlled resource for your workspace.\n",
+    "\n",
+    "Widget input parameters include:\n",
+    "\n",
+    "- `Access`: Defaults to `SHARED_ACCESS`, which grants bucket access to all users with access to the workspace, based on the user's workspace access (Reader or Writer). `PRIVATE_ACCESS` ensures the bucket and its contents cannot be shared.\n",
+    "- `Auto-delete`: Number of days after which to auto-delete the objects in the bucket. This option is a shortcut for specifying a [lifecycle rule](https://github.com/DataBiosphere/terra-cli#gcs-bucket-lifecycle-rules). If no value is provided,\n",
```

Comment:
Looks like you need to complete this sentence

Reply:
fixed!